maria-oss/src/cli.ts

```
@@ -0,0 +1,69 @@
+export interface CLIOptions {
+  model?: string;
+  debug?: boolean;
+}
+
+export function createCLI(): Command {
+  const program = new Command();
+
+  program
+    .name('maria')
+    .description('MARIA - AI-Powered Development Platform')
+    .version('1.0.7');
```

ESLint

[error] Replace `⏎····.name('maria')⏎····.description('MARIA·-·AI-Powered·Development·Platform')⏎····` with `.name('maria').description('MARIA·-·AI-Powered·Development·Platform')`

(prettier/prettier)